### DISCUSSION TITLE
Hebrew support

### DISCUSSION THREAD

**Discussion by humy66** [deepgram/community#219] [Feature Requests] [answered]
I am working on the promotional $200 budget.
Trying to connect to the wss://api.deepgram.com/v1/listen endpoint fails when asking for 
1) whisper - wss://api.deepgram.com/v1/listen?model=whisper
2) Hebrew - wss://api.deepgram.com/v1/listen?language=he
However works fine for 
wss://api.deepgram.com/v1/listen - transcribing english

Any ideas ?

**Comment by nikolawhallon** [comment, marked as answer]
Currently, Hebrew is only supported via Whisper, which is batch-only and not supported for streaming - I think that's what's going on here!

**Reply by jaxomlotus** [reply]
Is there any information on when/if Hebrew support will be added to Nova?

**Reply by aviadr1** [reply]
There is now a free (even for commerical use) dataset for hebrew model training, prehaps that will speed up adding hebrew to nova?
https://github.com/orgs/deepgram/discussions/903

**Comment by jpvajda** [comment]
relates to: https://github.com/orgs/deepgram/discussions/903

**Comment by aviadr1** [comment]
the Ivrit team is happy to provide any other resources to make hebrew well supported both in opensource and commercial products like deepgram.

The non profit ivrit.ai org is providing high quality dataset with completely free license that is also applicable to commerical use.
dataset & trained whisper model: https://huggingface.co/ivrit-ai
license: https://www.ivrit.ai/en/the-license/

**Comment by aszmyd** [comment]
Our product would waits for having Hebrew support in streaming with nova. Is there any timeframe when that could happend?

**Comment by davidstrouk** [comment]
I would also be interested in Hebrew support for streaming with Nova. Are there any updates on when this might be available? Thanks!

**Comment by deman311** [comment]
There are a lot of people waiting for this!

**Comment by thewh1teagle** [comment]
Waiting for this!

**Comment by mercuryyy** [comment]
How is deepgram still not supporting hebrew ?

**Comment by jaxomlotus** [comment]
@team-deepgram. 

I saw an announcement today that you have released 35 new languages, I was disappointed that Hebrew is still not included and would be grateful for you considering adding it. Thank you
